databroker/glue.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, division, print_function
+
+from glue.core.data import Data, Component
+
+
+def read_header(header):
+    out = []
+    for stream in header.stream_names:
+        result = Data(label="{stream}_{uid}".format(stream=stream, uid=header.start['uid']))
+        tbl = header.table(stream)
```

Comment:
should have `fill=True` maybe? (Or should header be assumed filled before entering?)

Reply:
Yes, it seems reasonable to have all the fields filled, though I'm not sure in which cases we may get empty fields. Do you have an example?

Reply:
We discussed it with @tacaswell and decided `fill=True` should be there.